.github/workflows/test.yml

```diff
@@ -157,7 +157,8 @@ jobs:
           file_system: ${{ matrix.file_system }}
           tpm_enabled: true
           suite: "eve-upgrade.tests.txt"
-          eve_image: ${{ inputs.eve_image }}
+          eve_image: lfedge/eve:11.0.5-lts
```

Comment:
Inside the test [workflow](https://github.com/lf-edge/eden/blob/master/tests/workflow/eve-upgrade.tests.txt) we run several steps:

1. Run and onboard EVE-OS
2. Run update of baseOS image from http store
3. Downgrade to the original baseOS
4. Run update of baseOS image from oci store
5. Downgrade to the original baseOS

In steps 2 and 4 we use different versions of baseOS to not allow EVE-OS just to check another partition and boot from it without downloading and writing. Please do not forget it (for example we can split oci and http different fresh workers).
We use downgrade as we still want to check an ability of current (testing) version to be updated.